context/brand-context/springernature/scss/10-settings/buttons.scss

```diff
@@ -1,17 +1,19 @@
 
- /**
+/**
  *  @springernature/global-button
  *  Button settings
  *
  */
 
-$context--button-border-radius: 26px;
+$context--button-border-radius: 32px;
+$context--button-border-width-style: 4px solid;
 $context--button-font-size-normal: 1.4rem;
-$context--button-padding: 10px 35px;
+$context--button-padding: 16px 32px;
 $context--button-padding-large: 10px 35px;
 $context--button-letter-spacing: 0.7px;
-$context--button-font-family: $context--font-family-sans;
-$context--button-transition: background-color 0.2s;
+$context--button-font-family: $font-family-sans;
```

Comment:
Preserve $context--font-family-sans as a val

Reply:
You beat us to it! :)